gnovm/pkg/gnolang/helpers.go

```diff
@@ -26,6 +27,11 @@ func IsRealmPath(pkgPath string) bool {
 		!ReGnoRunPath.MatchString(pkgPath)
 }
 
+// IsPackagePath determines whether the given pkgpath is for a package.
+func IsPackagePath(pkgPath string) bool {
```

Comment:
could be confusing too use a function that check if pkgPath is a PackagePath ?
Don't know what should i do

Reply:
> "PurePackagePath"? if you mean the name of the function.

I like it: [913e440](https://github.com/gnolang/gno/pull/3042/commits/913e440d2635253039b94e3e1f153350ca29fe5b)